### DISCUSSION TITLE
pdf editor template, no content!

### DISCUSSION THREAD

**Discussion by MixTecnico** [YetiForceCompany/YetiForceCRM#17045] [Ignored/Incomplete]
My system works perfectly, but I needed to update a spreadsheet that is generated through a template, but I see that nothing loads!

Is it some limit or permission?
or something related to the wysiwyg editor plugin?

any ideas?

attached video!

https://user-images.githubusercontent.com/29989839/234363913-2f488b0c-7ee8-4ee3-958c-02e44f45ddef.mp4

**Comment by mariuszkrzaczkowski** [comment]
https://github.com/YetiForceCompany/YetiForceCRM/issues/13908

bug/error >> https://github.com/YetiForceCompany/YetiForceCRM/issues/new?assignees=&labels=&template=1-bug-report.md&title=%5Bbug%5D+
question >> https://github.com/YetiForceCompany/YetiForceCRM/discussions

**Comment by mariuszkrzaczkowski** [comment]
check your browser console and server logs

**Reply by MixTecnico** [reply]
I checked the logs, but there are no errors regarding the problem...

is there a way to reload the wysiwyg plugin

**Reply by mariuszkrzaczkowski** [reply]
no, there must be some mistake

**Reply by MixTecnico** [reply]
I was checking in the browser, I see that it uses ckeditor...

I have some errors with it! but from the log there are no problems!

![image](https://user-images.githubusercontent.com/29989839/234591274-0b789616-2eff-4031-8896-8dff81d44395.png)

**Reply by MixTecnico** [reply]
> no, there must be some mistake

I was able to solve, reload the ckeditor library, and everything is 100% production!

Thank you also for answering!
